x-pack/plugins/reporting/server/routes/internal/diagnostic/browser.ts

```diff
@@ -34,6 +34,14 @@ const logsToHelpMapFactory = (docLinks: DocLinksServiceSetup) => ({
     defaultMessage: `Unable to use Chromium sandbox. This can be disabled at your own risk with 'xpack.screenshotting.browser.chromium.disableSandbox'. Please see {url}`,
     values: { url: docLinks.links.reporting.browserSandboxDependencies },
   }),
+
+  'Fontconfig error: Cannot load default config file': i18n.translate(
```

Comment:
Can you add a test case in `x-pack/plugins/reporting/server/routes/internal/diagnostic/integration_tests/browser.test.ts`?

Reply:
added it in